src/ring_theory/fractional_ideal.lean

```diff
@@ -308,7 +304,7 @@ end
 
 lemma fractional_inf (I J : fractional_ideal S P) : is_fractional S (I.1 ⊓ J.1) :=
```

Comment:
Since you have `val_eq_coe` above, it probably makes sense to write this
```suggestion
lemma fractional_inf (I J : fractional_ideal S P) : is_fractional S (I ⊔ J : submodule R P) :=
```
It's unfortunate the elaborator isn't able to work out the coercion itself.